doc/getting_started.rst

```diff
@@ -18,7 +18,7 @@ defined in the anaconda-project file:
 
 .. code:: bash
 
-   conda install anaconda-project tornado<5.0
+   conda install anaconda-project
```

Comment:
Depending on what else is installed, this won't necessarily get the latest version.  Should we specify the version here just in case?

Reply:
I suppose. I was trying to make it as minimal as possible, but better to be safe